prime-router/metadata/fhir_mapping/hl7/segments/PID/PD1Extension.yml

```diff
@@ -1,6 +1,6 @@
 url:
   type: SYSTEM_URL
-  value: pd1-patient
+  value: pd1-patient-additional-demographic
```

Comment:
Why add this on to the end? Feels like it breaks the pattern a bit

Reply:
The pattern I've update across the board is now that the the name here lines up exactly with the name of the HL7 segment that it's storing data from since it could get used to store PD1 data not on Patient